server/viame_server/viame_detection.py

```diff
@@ -55,7 +55,7 @@ def _get_clip_meta(self, folder):
             'videoUrl': videoUrl,
         }
 
-    @access.public(scope=TokenScope.DATA_READ, cookie=True)
+    @access.user
```

Comment:
Why does this require user login, but the `export_detections` endpoint below is public?

Reply:
This endpoint returns JSON meant to be ajax'd on the page.

The `export_detections` endpoint is a download link, so the browser must put it in a `href` with `target=_blank` to open it in a new tab and trigger the download.  Naturally you can't set headers with an href, so you have to use cookie authentication, which is generally unsafe (CSRF) but works fine when you're doing read-only operations like download.

Public just means no authorization header, but you'll still 403 if you try to download it without a cookie for a user without read access.

All the existing girder views (item, file) work like this when you use `/<model>/download`.